### ISSUE TITLE
node.el.detach() breaks focused elements

### ISSUE THREAD

**Issue by mgwidmann** [troolee/gridstack.js#46]
This line here:
https://github.com/troolee/gridstack.js/blob/master/src/gridstack.js#L675

After calling `resize()` on an element which has a text field which the user is typing into, detaching and appending again causes the user to lose focus.

Why is `detach()` called and is there any other way this can be done? Specifically, I am using a rich text editor inside the node and detaching and appending again breaks the editor.

Commenting these lines out fixes my problem, but I'm not sure what problems it will cause.

**Comment by troolee** [comment]
This code is sorting nodes in DOM so if you resize your browser and the grid collapses to single-column mode the items will appear in correct order. 

I will check if `detach` is needed there.

**Comment by mgwidmann** [comment]
Is there a way we can pass in a flag to resize to turn that off?

**Comment by troolee** [comment]
I will check if detaching is necessary there. Maybe I will be able to sort nodes without detaching.

**Comment by ishields** [comment]
I think no matter how you rearrange the nodes focus would be lost.  But it seems that not all use cases for grid stack will require a "responsive" view, so maybe it makes sense to add an "allow_responsive" flag or something to avoid this problem for those cases.  This would only be a good solution if not rearranging the nodes doesn't have any other negative affects.

**Comment by troolee** [comment]
Good point. I need to rearrange nodes for one-column view only.

**Comment by troolee** [comment]
Should be fine now.
